<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pdfHeaderEdit</class>
 <widget class="QWidget" name="pdfHeaderEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>660</width>
     <height>121</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>181</y>
     <width>661</width>
     <height>171</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>160</width>
     <height>112</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="V3">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>单位名称：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="companyEdit">
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="text">
       <string>江苏兰菱（海安）机电有限公司</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>副标题：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="subTitleEdit">
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>40</y>
     <width>160</width>
     <height>182</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="V1">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>电话：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>传真：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>电子信箱：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>网址：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>160</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="V2">
    <item>
     <widget class="QLineEdit" name="telEdit">
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="faxEdit">
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="mailEdit">
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="websiteEdit">
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="text">
       <string>lanlingjidian.cn.gongchang.com/</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>501</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>规格：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="sizeEdit">
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>型号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="typeNoEdit">
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>编号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="NoEdit">
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="myButton" name="exportButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>360</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>导出曲线报表</string>
   </property>
  </widget>
  <widget class="myCheckBox" name="check1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>310</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>转速图</string>
   </property>
  </widget>
  <widget class="myCheckBox" name="check2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>310</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>转矩图</string>
   </property>
  </widget>
  <widget class="myCheckBox" name="check3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>310</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>功率图</string>
   </property>
  </widget>
  <widget class="myButton" name="exportDataChartButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>360</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>导出数据报表</string>
   </property>
  </widget>
  <widget class="QLabel" name="ret">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>410</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>155</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myButton</class>
   <extends>QPushButton</extends>
   <header>mybutton.h</header>
  </customwidget>
  <customwidget>
   <class>myCheckBox</class>
   <extends>QCheckBox</extends>
   <header>mycheckbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>subTitleEdit</tabstop>
  <tabstop>telEdit</tabstop>
  <tabstop>faxEdit</tabstop>
  <tabstop>mailEdit</tabstop>
  <tabstop>websiteEdit</tabstop>
  <tabstop>sizeEdit</tabstop>
  <tabstop>typeNoEdit</tabstop>
  <tabstop>NoEdit</tabstop>
  <tabstop>check1</tabstop>
  <tabstop>check2</tabstop>
  <tabstop>check3</tabstop>
  <tabstop>exportButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
